content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -127,7 +127,7 @@ virtual resource type would be used if that becomes necessary.
 
 ## HTTP media types {#alternate-representations-of-resources}
 
-Over HTTP, Kubernetes supports JSON and Protobuf wire encodings.
+Over HTTP, Kubernetes supports JSON, YAML, CBOR and Protobuf wire encodings.
```

Comment:
I agree about mentioning CBOR being alpha

```suggestion
Over HTTP, Kubernetes supports JSON, YAML, CBOR (alpha) and Protobuf wire encodings.
```

Reply:
On line 341 we have whole section for CBOR that doesn't mention alpha at all, only the feautre gate CBORServingAndStorage. Why mention Alpha here and not there?